src/main/kotlin/org/malachite/estella/services/SecurityService.kt

```diff
@@ -0,0 +1,77 @@
+package org.malachite.estella.services
+
+import io.jsonwebtoken.Jwts
+import io.jsonwebtoken.SignatureAlgorithm
+import org.malachite.estella.commons.models.people.User
+import org.malachite.estella.people.domain.UserRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import java.sql.Date
+import javax.servlet.http.Cookie
+import javax.servlet.http.HttpServletResponse
+
+@Service
+class SecurityService(@Autowired val userRepository: UserRepository) {
+
+    private val authSecret = "secret"
```

Comment:
Te secrety to tak na teraz tutaj, czy one na stałe i bawimy się w IO? XD

Reply:
Tak na teraz tutaj